index.html

```diff
@@ -6,6 +6,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+    <link rel="stylesheet" type="text/css" href="css/Grid.css">
```

Comment:
What is the need of a new Grid.css file we can use the bootstrap grids itself so it is not needed.

Reply:
Sorry sir it was ...for the suggestion form I will edit that

Reply:
okay